src/flow/ext_multiroute_flow.jl

```diff
@@ -181,7 +181,7 @@ function slope{T<:AbstractFloat}(
   )
   slope = 0
   for e in edges(flow_graph)
-    if cut[dst(e)] - cut[src(e)] > 0 &&
+    if cut[dst(e)] == 2 > cut[src(e)] &&
```

Comment:
For the record : http://docs.julialang.org/en/release-0.5/manual/mathematical-operations/#chaining-comparisons